src/target/riscv/riscv-013.c

```diff
@@ -499,9 +503,11 @@ static dmi_status_t dmi_scan(struct target *target, uint32_t *address_in,
 
 	assert(info->abits != 0);
 
+	const uint32_t address_out_with_dbgbase = address_out + target->dbgbase;
```

Comment:
Seems unnecessary here, especially given it's used only once.

Reply:
done